catalog/spatial/csw/spatial-csw-endpoint/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/mappings/TestCswRecordMapperFilterVisitor.java

```diff
@@ -259,6 +263,38 @@ public void testVisitPropertyIsEqualTo() {
     }
 
     @Test
+    public void testVisitPropertyIsEqualToFunction() {
+        Expression val = factory.literal(true);
+        ExtendedGeotoolsFunctionFactory geotoolsFunctionFactory =
+                new ExtendedGeotoolsFunctionFactory();
+        Function function = geotoolsFunctionFactory.function("proximity",
+                Arrays.asList(attrExpr, factory.literal(1), factory.literal("Marry little")),
+                null);
+
+        PropertyIsEqualTo filter = factory.equals(function, val);
+
+        PropertyIsEqualTo duplicate = (PropertyIsEqualTo) visitor.visit(filter, null);
+        assertThat(duplicate.getExpression1(), Matchers.instanceOf(Function.class));
+        Function dupFunction = (Function) duplicate.getExpression1();
+        assertThat(dupFunction.getFunctionName(), is(function.getFunctionName()));
+        assertThat(dupFunction.getParameters()
```

Comment:
You don't need these assertions since your `assertThat(dupFunction.getParameters(), is(function.getParameters()))` will check that the lists are equal (and therefore that the corresponding pairs of elements in the two lists are equal).